paf-mvp-pattern-library-audit/stories/components/AdvertStatus.js

```diff
@@ -0,0 +1,63 @@
+import Button from './Button';
+import { Attention, Warning, ArrowRight, Check, External } from './Icons';
+
+const content = {
```

Comment:
Couldn't this be handled with language bundles, as for the CMP?

Reply:
See [here](https://github.com/prebid/paf-mvp-implementation/pull/180#discussion_r896836958).